src/api/spec/controllers/webui/project/public_key_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe Webui::Project::PublicKeyController, type: :controller, vcr: true do
```

Comment:
As you don't add cassettes do you really need vcr: true?
https://github.com/vcr/vcr